ksql-engine/src/test/java/io/confluent/ksql/SchemaTranslationTest.java

```diff
@@ -47,7 +47,7 @@ public SchemaTranslationTest(final String name, final Query query) {
 
   @Test
   public void shouldBuildAndExecuteQueries() {
-    EndToEndEngineTestUtil.shouldBuildAndExecuteQuery(this.query);
+    EndToEndEngineTestUtil.shouldBuildAndExecuteQuery(this.query, "none");
```

Comment:
I simply added a hard-coded value here with optimizations turned off.  My thinking is just adding the parameter was good enough instead refactoring out a separate method at this point.

Reply:
If we test with optimization turned on, do we need to manually change this value in the test code and re-run?